components/Home/__snapshots__/index.test.tsx.snap

```diff
@@ -154,7 +154,7 @@ exports[`Msig Home it renders the vesting balance, available balance, and msig a
   >
     <a
       class="c5"
-      href="/withdraw"
+      href="/withdraw?network=t"
```

Comment:
Not sure where this comes from, but shouldn't the network query arg be the full network name instead of the cointype?

Reply:
These are hardcoded strings in the zondax library that (unfortunately) didn't line up with our naming (besides mainnet). This should operate more like "coinType" - its EITHER `mainnet` or `calibrationnet`, so i don't think we should use the NetworkName for this